samples/snippets/prediction_service/predict_text_classification_single_label_sample.py

```diff
@@ -45,7 +45,7 @@ def predict_text_classification_single_label_sample(
     )
     print("response")
     print(" deployed_model_id:", response.deployed_model_id)
-    # See gs://google-cloud-aiplatform/schema/predict/prediction/text_classification.yaml for the format of the predictions.
+
```

Comment:
Missing updated comment here to point to new storage blob

Reply:
There is no YAML prediction output schema for text classification in the bucket, so I went ahead and removed the comment altogether:
```shell
gsutil ls gs://google-cloud-aiplatform/schema/predict/prediction/
```

Reply:
Got it!